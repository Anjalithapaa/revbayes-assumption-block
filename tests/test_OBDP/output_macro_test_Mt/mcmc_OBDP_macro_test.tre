#NEXUS

	Begin taxa;
	Dimensions ntax=55;
	Taxlabels
		f0
		f1
		f10
		f11
		f12
		f13
		f14
		f15
		f16
		f17
		f18
		f19
		f2
		f20
		f21
		f22
		f23
		f24
		f25
		f26
		f27
		f28
		f29
		f3
		f30
		f31
		f32
		f33
		f34
		f35
		f36
		f37
		f38
		f39
		f4
		f40
		f41
		f42
		f43
		f44
		f45
		f46
		f47
		f48
		f49
		f5
		f50
		f51
		f52
		f53
		f54
		f6
		f7
		f8
		f9
		;
End;

Begin trees;
tree TREE1 = [&R](((((((((((f33[&index=18]:2.390144,f39[&index=17,age_95%_HPD={1.207,1.207}]:1.183143)[&index=56,posterior=1.000000,age_95%_HPD={2.39014,2.39014}]:1.643292,f14[&index=16]:4.033436)[&index=57,posterior=1.000000,age_95%_HPD={4.03344,4.03344}]:0.203841,(f37[&index=14,age_95%_HPD={0.811,0.811}]:1.254233,f6[&index=15]:2.065233)[&index=58,posterior=1.000000,age_95%_HPD={2.06523,2.06523}]:2.172044)[&index=59,posterior=1.000000,age_95%_HPD={4.23728,4.23728}]:0.430217,(f1[&index=13]:4.410424,f48[&index=12,age_95%_HPD={2.92,2.92}]:1.490423)[&index=60,posterior=1.000000,age_95%_HPD={4.41042,4.41042}]:0.257070)[&index=61,posterior=1.000000,age_95%_HPD={4.66749,4.66749}]:0.036265,f52[&index=11,age_95%_HPD={3.948,3.948}]:0.755758)[&index=62,posterior=1.000000,age_95%_HPD={4.70376,4.70376}]:0.124161,((((f2[&index=21]:0.262547,f22[&index=22]:0.262547)[&index=63,posterior=1.000000,age_95%_HPD={0.262547,0.262547}]:0.257095,f9[&index=20]:0.519642)[&index=64,posterior=1.000000,age_95%_HPD={0.519642,0.519642}]:2.477764,f31[&index=19]:2.997406)[&index=65,posterior=1.000000,age_95%_HPD={2.99741,2.99741}]:1.151833,f3[&index=23]:4.149239)[&index=66,posterior=1.000000,age_95%_HPD={4.14924,4.14924}]:0.678681)[&index=67,posterior=1.000000,age_95%_HPD={4.82792,4.82792}]:0.089269,(((((f19[&index=10]:0.960618,f35[&index=9,age_95%_HPD={0.369001,0.369001}]:0.591617)[&index=68,posterior=1.000000,age_95%_HPD={0.960618,0.960618}]:2.447214,(f32[&index=8]:1.364175,f34[&index=7,age_95%_HPD={0.325002,0.325002}]:1.039173)[&index=69,posterior=1.000000,age_95%_HPD={1.36418,1.36418}]:2.043657)[&index=70,posterior=1.000000,age_95%_HPD={3.40783,3.40783}]:1.020853,(f44[&index=5,age_95%_HPD={1.636,1.636}]:1.091528,f47[&index=6,age_95%_HPD={2.396,2.396}]:0.331528)[&index=71,posterior=1.000000,age_95%_HPD={2.72753,2.72753}]:1.701156)[&index=72,posterior=1.000000,age_95%_HPD={4.42869,4.42869}]:0.325121,f50[&index=4,age_95%_HPD={3.114,3.114}]:1.639805)[&index=73,posterior=1.000000,age_95%_HPD={4.75381,4.75381}]:0.021998,(f51[&index=2,age_95%_HPD={3.803,3.803}]:0.163454,f7[&index=3]:3.966455)[&index=74,posterior=1.000000,age_95%_HPD={3.96646,3.96646}]:0.809349)[&index=75,posterior=1.000000,age_95%_HPD={4.7758,4.7758}]:0.141385)[&index=76,posterior=1.000000,age_95%_HPD={4.91719,4.91719}]:0.196113,((((((f13[&index=28]:0.124530,f5[&index=29]:0.124530)[&index=77,posterior=1.000000,age_95%_HPD={0.12453,0.12453}]:0.899592,f12[&index=30]:1.024122)[&index=78,posterior=1.000000,age_95%_HPD={1.02412,1.02412}]:0.789505,((f23[&index=25]:0.850471,f29[&index=26]:0.850471)[&index=79,posterior=1.000000,age_95%_HPD={0.850471,0.850471}]:0.879540,f15[&index=27]:1.730011)[&index=80,posterior=1.000000,age_95%_HPD={1.73001,1.73001}]:0.083616)[&index=81,posterior=1.000000,age_95%_HPD={1.81363,1.81363}]:0.078573,f11[&index=31]:1.892200)[&index=82,posterior=1.000000,age_95%_HPD={1.8922,1.8922}]:2.240334,((((f16[&index=39]:0.853935,f24[&index=40]:0.853935)[&index=83,posterior=1.000000,age_95%_HPD={0.853935,0.853935}]:1.965693,f46[&index=38,age_95%_HPD={2.261,2.261}]:0.558626)[&index=84,posterior=1.000000,age_95%_HPD={2.81963,2.81963}]:0.903915,((f21[&index=36]:0.042301,f27[&index=37]:0.042301)[&index=85,posterior=1.000000,age_95%_HPD={0.042301,0.042301}]:0.353652,f10[&index=35]:0.395953)[&index=86,posterior=1.000000,age_95%_HPD={0.395953,0.395953}]:3.327590)[&index=87,posterior=1.000000,age_95%_HPD={3.72354,3.72354}]:0.004675,((f18[&index=34]:2.203776,f25[&index=33]:2.203776)[&index=88,posterior=1.000000,age_95%_HPD={2.20378,2.20378}]:1.304805,f38[&index=32,age_95%_HPD={1.095,1.095}]:2.413579)[&index=89,posterior=1.000000,age_95%_HPD={3.50858,3.50858}]:0.219637)[&index=90,posterior=1.000000,age_95%_HPD={3.72822,3.72822}]:0.404316)[&index=91,posterior=1.000000,age_95%_HPD={4.13253,4.13253}]:0.064932,f43[&index=24,age_95%_HPD={1.383,1.383}]:2.814465)[&index=92,posterior=1.000000,age_95%_HPD={4.19747,4.19747}]:0.915836)[&index=93,posterior=1.000000,age_95%_HPD={5.1133,5.1133}]:0.039669,(f20[&index=41]:2.502598,f41[&index=42,age_95%_HPD={1.28,1.28}]:1.222597)[&index=94,posterior=1.000000,age_95%_HPD={2.5026,2.5026}]:2.650373)[&index=95,posterior=1.000000,age_95%_HPD={5.15297,5.15297}]:0.137621,f54[&index=1,age_95%_HPD={5.161,5.161}]:0.129591)[&index=96,posterior=1.000000,age_95%_HPD={5.29059,5.29059}]:0.002802,(((((((f17[&index=48]:0.421229,f26[&index=47]:0.421229)[&index=97,posterior=1.000000,age_95%_HPD={0.421229,0.421229}]:2.639045,f30[&index=46]:3.060274)[&index=98,posterior=1.000000,age_95%_HPD={3.06027,3.06027}]:0.331571,f0[&index=49]:3.391845)[&index=99,posterior=1.000000,age_95%_HPD={3.39185,3.39185}]:0.855983,f36[&index=45,age_95%_HPD={0.492,0.492}]:3.755828)[&index=100,posterior=1.000000,age_95%_HPD={4.24783,4.24783}]:0.056495,f49[&index=44,age_95%_HPD={3.097,3.097}]:1.207323)[&index=101,posterior=1.000000,age_95%_HPD={4.30432,4.30432}]:0.301657,f53[&index=43,age_95%_HPD={4.2,4.2}]:0.405979)[&index=102,posterior=1.000000,age_95%_HPD={4.60598,4.60598}]:0.431832,((((f28[&index=55]:2.685783,f45[&index=54,age_95%_HPD={1.748,1.748}]:0.937782)[&index=103,posterior=1.000000,age_95%_HPD={2.68578,2.68578}]:0.872403,f4[&index=53]:3.558186)[&index=104,posterior=1.000000,age_95%_HPD={3.55819,3.55819}]:0.081360,(f40[&index=52,age_95%_HPD={1.253,1.253}]:1.787163,f8[&index=51]:3.040163)[&index=105,posterior=1.000000,age_95%_HPD={3.04016,3.04016}]:0.599383)[&index=106,posterior=1.000000,age_95%_HPD={3.63955,3.63955}]:1.227408,f42[&index=50,age_95%_HPD={1.285,1.285}]:3.581954)[&index=107,posterior=1.000000,age_95%_HPD={4.86695,4.86695}]:0.170858)[&index=108,posterior=1.000000,age_95%_HPD={5.03781,5.03781}]:0.255582)[&index=109,posterior=1.000000,age_95%_HPD={5.29339,5.29339}]:0.000000;
End;
