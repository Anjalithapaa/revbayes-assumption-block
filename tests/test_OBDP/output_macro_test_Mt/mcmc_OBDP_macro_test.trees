Iteration	Posterior	Likelihood	Prior	obd_tree
0	-53841.3	-53768.01	-73.29724	(f54[&index=10]:0.029247,((f53[&index=9]:0.665039,(f2[&index=8]:4.251894,((f0[&index=7]:2.738184,f3[&index=6]:2.738184)[&index=11]:1.431589,(f50[&index=5]:0.256827,f4[&index=4]:3.370827)[&index=12]:0.798946)[&index=13]:0.082122)[&index=14]:0.613144)[&index=15]:0.296036,(f52[&index=3]:0.444280,(f1[&index=2]:4.333303,f51[&index=1]:0.530303)[&index=16]:0.058977)[&index=17]:0.768794)[&index=18]:0.029173)[&index=19]:0.000000;
10	-38797.17	-38764.23	-32.93594	(((((f3[&index=6]:3.114000)f50[&index=12]:0.059353,f4[&index=4]:3.173353)[&index=13]:0.774647)f52[&index=17]:1.122518,(f53[&index=9]:0.849068,(((f0[&index=7]:0.687329,f2[&index=8]:0.687329)[&index=11]:0.558570,f1[&index=2]:1.245899)[&index=14]:2.557101)f51[&index=16]:1.246068)[&index=15]:0.021450)[&index=18]:0.090482)f54[&index=19]:0.000000;
20	-37089.18	-37063.83	-25.34947	(((((f0[&index=7]:0.228984,f2[&index=8]:0.228984)[&index=11]:0.074659,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.593496,(f51[&index=1]:0.674128,(((f3[&index=6]:1.310398,f4[&index=4]:1.310398)[&index=13]:1.803602)f50[&index=12]:0.834000)f52[&index=17]:0.529128)[&index=16]:0.316368)[&index=18]:0.367504)f54[&index=19]:0.000000;
30	-36705.26	-36682.51	-22.75018	(((((f0[&index=7]:0.148345,f2[&index=8]:0.148345)[&index=11]:0.155298,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.513943,(((f3[&index=6]:1.711986,f4[&index=4]:1.711986)[&index=13]:2.091014)f51[&index=12]:0.787600,(f50[&index=5]:0.834000)f52[&index=17]:0.642600)[&index=16]:0.123343)[&index=18]:0.447057)f54[&index=19]:0.000000;
40	-36498.86	-36474.72	-24.14007	(((((f0[&index=7]:0.148345,f2[&index=8]:0.148345)[&index=11]:0.155298,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.513943,(((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.155576)f51[&index=12]:0.505497,(f50[&index=5]:0.834000)f52[&index=17]:0.360497)[&index=16]:0.405447)[&index=18]:0.447057)f54[&index=19]:0.000000;
50	-36401.36	-36376.53	-24.83555	((((f50[&index=5]:0.834000)f52[&index=17]:0.222487,((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.155576)f51[&index=12]:0.367487)[&index=16]:0.280703,(((f0[&index=7]:0.155872,f2[&index=8]:0.155872)[&index=11]:0.147770,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.251190)[&index=18]:0.709810)f54[&index=19]:0.000000;
60	-36230.58	-36200.54	-30.04058	(((f50[&index=5]:0.973717,(f51[&index=1]:0.270497,((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.300576)f52[&index=17]:0.125497)[&index=12]:0.014220)[&index=16]:0.257006,(((f0[&index=7]:0.152301,f2[&index=8]:0.152301)[&index=11]:0.151342,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.144723)[&index=18]:0.816277)f54[&index=19]:0.000000;
70	-36161.26	-36134.16	-27.09963	((((f50[&index=5]:0.996711,((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.300576)f52[&index=17]:0.162711)[&index=12]:0.029399,f51[&index=1]:0.337110)[&index=16]:0.191536,(((f0[&index=7]:0.152301,f2[&index=8]:0.152301)[&index=11]:0.151342,f1[&index=2]:0.303643)[&index=14]:3.896357)f53[&index=15]:0.131646)[&index=18]:0.829354)f54[&index=19]:0.000000;
80	-36124.21	-36092.51	-31.70078	((((f50[&index=5]:0.892152,((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.300576)f52[&index=17]:0.058152)[&index=12]:0.047693,f51[&index=1]:0.250845)[&index=16]:0.193734,(((f0[&index=7]:0.152301,f2[&index=8]:0.152301)[&index=11]:0.151342,f1[&index=2]:0.303643)[&index=14]:3.896895)f53[&index=15]:0.047042)[&index=18]:0.913421)f54[&index=19]:0.000000;
90	-36077.64	-36048.34	-29.29183	((((f51[&index=1]:0.219629,f50[&index=5]:0.908629)[&index=12]:0.001440,((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.300576)f52[&index=17]:0.076068)[&index=16]:0.226159,(((f0[&index=7]:0.152301,f2[&index=8]:0.152301)[&index=11]:0.151342,f1[&index=2]:0.303643)[&index=14]:3.896895)f53[&index=15]:0.049691)[&index=18]:0.910772)f54[&index=19]:0.000000;
100	-36064.91	-36036.58	-28.33668	(((((f3[&index=6]:1.647424,f4[&index=4]:1.647424)[&index=13]:2.300576)f52[&index=17]:0.053400,(f50[&index=5]:0.882540,f51[&index=1]:0.193540)[&index=12]:0.004860)[&index=16]:0.264087,(((f0[&index=7]:0.155767,f2[&index=8]:0.155767)[&index=11]:0.147876,f1[&index=2]:0.303643)[&index=14]:3.896895)f53[&index=15]:0.064950)[&index=18]:0.895513)f54[&index=19]:0.000000;
