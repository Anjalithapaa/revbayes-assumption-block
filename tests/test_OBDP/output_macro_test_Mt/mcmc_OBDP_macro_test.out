Iteration	alpha_morpho	alpha_seq	branch_rates	clock_morpho	er	lambda	mu	obd_tree	omega	psi	sf	start_time
0	0.534204	0.4523303	0.0001074664	0.4202539	[0.110005,0.0417276,0.0818335,0.200507,0.409049,0.156878]	0.007298359	0.011636	((((((f28[&index=55]:2.685782,f45[&index=54]:0.937782)[&index=56]:0.872403,f4[&index=53]:3.558185)[&index=57]:0.081360,(f40[&index=52]:1.787163,f8[&index=51]:3.040163)[&index=58]:0.599383)[&index=59]:1.227408,f42[&index=50]:3.581954)[&index=60]:0.170858,((((f0[&index=49]:3.391845,((f17[&index=48]:0.421229,f26[&index=47]:0.421229)[&index=61]:2.639045,f30[&index=46]:3.060274)[&index=62]:0.331571)[&index=63]:0.855983,f36[&index=45]:3.755828)[&index=64]:0.056495,f49[&index=44]:1.207323)[&index=65]:0.301657,f53[&index=43]:0.405979)[&index=66]:0.431832)[&index=67]:0.255582,(((f41[&index=42]:1.222597,f20[&index=41]:2.502597)[&index=68]:2.650373,(((((((f24[&index=40]:0.853933,f16[&index=39]:0.853933)[&index=69]:1.965693,f46[&index=38]:0.558626)[&index=70]:0.903915,((f27[&index=37]:0.042299,f21[&index=36]:0.042299)[&index=71]:0.353652,f10[&index=35]:0.395950)[&index=72]:3.327590)[&index=73]:0.004675,((f18[&index=34]:2.203774,f25[&index=33]:2.203774)[&index=74]:1.304805,f38[&index=32]:2.413579)[&index=75]:0.219637)[&index=76]:0.404316,(f11[&index=31]:1.892199,((f12[&index=30]:1.024121,(f5[&index=29]:0.124529,f13[&index=28]:0.124529)[&index=77]:0.899592)[&index=78]:0.789505,(f15[&index=27]:1.730010,(f29[&index=26]:0.850470,f23[&index=25]:0.850470)[&index=79]:0.879540)[&index=80]:0.083616)[&index=81]:0.078573)[&index=82]:2.240334)[&index=83]:0.064932,f43[&index=24]:2.814465)[&index=84]:0.915836,(((f3[&index=23]:4.149238,(((f22[&index=22]:0.262547,f2[&index=21]:0.262547)[&index=85]:0.257095,f9[&index=20]:0.519641)[&index=86]:2.477764,f31[&index=19]:2.997405)[&index=87]:1.151833)[&index=88]:0.678681,(((((f33[&index=18]:2.390143,f39[&index=17]:1.183143)[&index=89]:1.643292,f14[&index=16]:4.033435)[&index=90]:0.203841,(f6[&index=15]:2.065233,f37[&index=14]:1.254233)[&index=91]:2.172044)[&index=92]:0.430217,(f1[&index=13]:4.410423,f48[&index=12]:1.490423)[&index=93]:0.257070)[&index=94]:0.036265,f52[&index=11]:0.755758)[&index=95]:0.124161)[&index=96]:0.089269,(((((f19[&index=10]:0.960617,f35[&index=9]:0.591617)[&index=97]:2.447214,(f32[&index=8]:1.364173,f34[&index=7]:1.039173)[&index=98]:2.043657)[&index=99]:1.020853,(f47[&index=6]:0.331528,f44[&index=5]:1.091528)[&index=100]:1.701156)[&index=101]:0.325121,f50[&index=4]:1.639805)[&index=102]:0.021998,(f7[&index=3]:3.966454,f51[&index=2]:0.163454)[&index=103]:0.809349)[&index=104]:0.141385)[&index=105]:0.196113)[&index=106]:0.039669)[&index=107]:0.137621,f54[&index=1]:0.129591)[&index=108]:0.002802)[&index=109]:0.000000;	0.406959	0.1150825	[0.179482,0.393565,0.0817965,0.345156]	8.490851
1	0.534204	0.4523303	0.0001074664	0.4202539	[0.110005,0.0417276,0.0818335,0.200507,0.409049,0.156878]	0.007298359	0.011636	((((((f28[&index=55]:2.685782,f45[&index=54]:0.937782)[&index=56]:0.872403,f4[&index=53]:3.558185)[&index=57]:0.081360,(f40[&index=52]:1.787163,f8[&index=51]:3.040163)[&index=58]:0.599383)[&index=59]:1.227408,f42[&index=50]:3.581954)[&index=60]:0.170858,((((f0[&index=49]:3.391845,((f17[&index=48]:0.421229,f26[&index=47]:0.421229)[&index=61]:2.639045,f30[&index=46]:3.060274)[&index=62]:0.331571)[&index=63]:0.855983,f36[&index=45]:3.755828)[&index=64]:0.056495,f49[&index=44]:1.207323)[&index=65]:0.301657,f53[&index=43]:0.405979)[&index=66]:0.431832)[&index=67]:0.255582,(((f41[&index=42]:1.222597,f20[&index=41]:2.502597)[&index=68]:2.650373,(((((((f24[&index=40]:0.853933,f16[&index=39]:0.853933)[&index=69]:1.965693,f46[&index=38]:0.558626)[&index=70]:0.903915,((f27[&index=37]:0.042299,f21[&index=36]:0.042299)[&index=71]:0.353652,f10[&index=35]:0.395950)[&index=72]:3.327590)[&index=73]:0.004675,((f18[&index=34]:2.203774,f25[&index=33]:2.203774)[&index=74]:1.304805,f38[&index=32]:2.413579)[&index=75]:0.219637)[&index=76]:0.404316,(f11[&index=31]:1.892199,((f12[&index=30]:1.024121,(f5[&index=29]:0.124529,f13[&index=28]:0.124529)[&index=77]:0.899592)[&index=78]:0.789505,(f15[&index=27]:1.730010,(f29[&index=26]:0.850470,f23[&index=25]:0.850470)[&index=79]:0.879540)[&index=80]:0.083616)[&index=81]:0.078573)[&index=82]:2.240334)[&index=83]:0.064932,f43[&index=24]:2.814465)[&index=84]:0.915836,(((f3[&index=23]:4.149238,(((f22[&index=22]:0.262547,f2[&index=21]:0.262547)[&index=85]:0.257095,f9[&index=20]:0.519641)[&index=86]:2.477764,f31[&index=19]:2.997405)[&index=87]:1.151833)[&index=88]:0.678681,(((((f33[&index=18]:2.390143,f39[&index=17]:1.183143)[&index=89]:1.643292,f14[&index=16]:4.033435)[&index=90]:0.203841,(f6[&index=15]:2.065233,f37[&index=14]:1.254233)[&index=91]:2.172044)[&index=92]:0.430217,(f1[&index=13]:4.410423,f48[&index=12]:1.490423)[&index=93]:0.257070)[&index=94]:0.036265,f52[&index=11]:0.755758)[&index=95]:0.124161)[&index=96]:0.089269,(((((f19[&index=10]:0.960617,f35[&index=9]:0.591617)[&index=97]:2.447214,(f32[&index=8]:1.364173,f34[&index=7]:1.039173)[&index=98]:2.043657)[&index=99]:1.020853,(f47[&index=6]:0.331528,f44[&index=5]:1.091528)[&index=100]:1.701156)[&index=101]:0.325121,f50[&index=4]:1.639805)[&index=102]:0.021998,(f7[&index=3]:3.966454,f51[&index=2]:0.163454)[&index=103]:0.809349)[&index=104]:0.141385)[&index=105]:0.196113)[&index=106]:0.039669)[&index=107]:0.137621,f54[&index=1]:0.129591)[&index=108]:0.002802)[&index=109]:0.000000;	0.406959	0.1150825	[0.179482,0.393565,0.0817965,0.345156]	8.490851
